.promu.e2e.yml

```diff
@@ -0,0 +1,24 @@
+go:
+  version: 1.17
+  cgo: true
+repository:
+  path: github.com/thanos-io/thanos
+build:
+  binaries:
+    - name: thanos
+      path: ./cmd/thanos
+  flags: -a -tags netgo -race
+  
+  ldflags: |
+    -X github.com/prometheus/common/version.Version={{.Version}}
+    -X github.com/prometheus/common/version.Revision={{.Revision}}
+    -X github.com/prometheus/common/version.Branch={{.Branch}}
+    -X github.com/prometheus/common/version.BuildUser={{user}}@{{host}}
+    -X github.com/prometheus/common/version.BuildDate={{date "20060102-15:04:05"}}
+crossbuild:
+  platforms:
+    - linux/amd64
+    - darwin/amd64
```

Comment:
Why do we need anything other than Linux for the e2e tests? All of the tests run inside of a container, which means that the will run on Linux, and thus need a Linux binary, right?

Reply:
Okay, makes sense , we can remove the other platforms for sure